docs/3-cloud-computing/3.2.6-eks.md

```diff
@@ -107,6 +107,17 @@ Now you will add an autoscaler to your cluster that will automatically scale you
 4. Verify that there is a `cluster-autoscaler` running in your `kube-system` namespace. Check the logs to see what it is doing.
 5. Check that the autoscaler successfully scales your Node Group down to 2 instances.
 
+## Exercise - Adding an Ingress controller to your cluster
+Finally we will add an Ingress controller so that you can access this cluster without having to use port-forwardding.
+1. Edit the `complete-demo.yaml` file so the front-end service is not outward facing and reapply the configuration.
+2. Install an Ingress controller that works with AWS.
```

Comment:
Lets specify that this should be the nginx ingress controller and give them a link with information.